map/bookmark_manager.cpp

```diff
@@ -1871,7 +1871,17 @@ void BookmarkManager::LoadBookmarkRoutine(std::string const & filePath, bool isT
     std::string fileSavePath = GetKMLPath(filePath);
     if (!fileSavePath.empty())
     {
-      auto kmlData = LoadKmlFile(fileSavePath, KmlFileType::Text);
+      auto const ext = base::GetFileExtension(filePath);
+      std::unique_ptr<kml::FileData> kmlData;
+      if (ext == ".kml")
```

Comment:
This line fixes broken loading of kmz files.
```suggestion
      if (ext == ".kml" || ext == ".kmz")
```

Reply:
thanks for suggestion, fixed